app/krs.js

```diff
@@ -61,7 +61,7 @@ const provisionMasterKey = co(function *(coin, customerId) {
   const keyType = process.config.supportedcoins[coin];
 
   const key = yield MasterKey.findOneAndUpdate({ coin: null, customerId: null, type: keyType },
-    { coin: coin, customerId: customerId, type: keyType });
+    { coin: coin, customerId: customerId, type: keyType }, { useFindAndModify: false });
```

Comment:
What is this change for?

Reply:
there was a deprecation warning for using `findOneAndUpdate`. Mongoose docs suggest setting `useFindAndModify` to `false` if you wish to stick to using `findOneAndUpdate` (default setting is `true` which means the `useFindAndModify` function would be used instead)